### PR TITLE
feat(login component for architect and client): login component share…

### PR DESCRIPTION
**Pull request by guardian-001**
## What does this do?

This PR introduces a shared login component that can be used by both architects and clients.

## Why did you do this?

The goal is to allow both architects and clients to log into their accounts using a unified component, thereby reducing code duplication.

## Who/what does this impact?

This change simplifies the development process for developers by minimizing the amount of code required.

### CODE REVIEW

File: src/modules/login/login.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+export type LoginFormProps = {
+  onSubmit?: SubmitHandler<FormType>;
+};
+export default function LoginShared({ onSubmit = () => {} }: LoginFormProps) {
+  return (
+    <KeyboardAvoidingView
+      behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
+      className={`flex-1  items-center justify-center  bg-secondary`}
+    >
+      <Stack.Screen
```
Comment (MohamedMegdich1) [inline src/modules/login/login.tsx:19]: Why are you using Stack.Screen here?

File: src/modules/shared/conditions.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            Linking.openURL('https://example.com');
+          }}
+          className={`text-xs font-bold text-primary-txt  `}
+        >
+          {translate('login.termsConditionsPart2')}
+        </Text>
+        {translate('login.termsConditionsPart3')}
+        <Text
+          onPress={() => {
+            Linking.openURL('https://example.com');
```
Comment (MohamedMegdich1) [inline src/modules/shared/conditions.tsx:25]: extract 'https://example.com to a environment variable

**Review by MohamedMegdich1 — CHANGES_REQUESTED** [review]

File: src/modules/login/login.tsx (hunk 1 above)
Comment (guardian-001) [reply src/modules/login/login.tsx:19]: image
image
i move it to _layout file

File: src/modules/shared/conditions.tsx (hunk 2 above)
Comment (guardian-001) [reply src/modules/shared/conditions.tsx]: image

done

**Pull request merged**